peakina/datapool.py

```diff
@@ -1,8 +1,8 @@
 from os import path
 from typing import TYPE_CHECKING, Any, Dict, Hashable, Optional
 
-from .cache import Cache
-from .datasource import DataSource
+from peakina.cache import Cache
```

Comment:
Why change this?

Reply:
To have an absolute import 🤔 
Is there an issue doing that ?

Reply:
No both are fine. Was just wondering if you had issues with testing or anything